Communication_test.py

```diff
@@ -0,0 +1,50 @@
+from machine import Pin
+import time
+
+red = Pin(15, Pin.OUT)
+green = Pin(14, Pin.OUT)
+redcount = 0
+greencount = 0
+
+def base2_light(s):
+    for i in s:
+        if i == '1':
+            green.toggle()
+            time.sleep(0.5)
+            green.toggle()
+            time.sleep(1)
+            redcount += 1
+        else:
+            red.toggle()
+            time.sleep(0.5)
+            red.toggle()
+            time.sleep(1)
+            greencount += 1
+    redcountans = int(input"How many times did the red light flash?")
+    if redcountans == redcount:
+        print("You're correct!")
+    else:
+        print("You're wrong!")
```

Comment:
the result of a wrong input should be that this character is shown in light signals again, not an output saying you are wrong.
```suggestion
        pass
```

Reply:
Didn't thought of that — thanks for the suggestion! I'll work on it.